bgpd/bgp_attr.c

```diff
@@ -2882,7 +2882,11 @@ size_t bgp_packet_mpattr_start(struct stream *s, struct peer *peer, afi_t afi,
 					    BGP_ATTR_NHLEN_IPV6_GLOBAL_AND_LL);
 				stream_put(s, &attr->mp_nexthop_global,
 					   IPV6_MAX_BYTELEN);
-				stream_put(s, &attr->mp_nexthop_local,
+				stream_put(s,
```

Comment:
I don't think we want to add the global nexthop twice.

Reply:
Yes, this is a wrong way to handle this. Leaving only one nexthop while len is for two breaks tests. Sri told they had the same issue reported and will upstream their patches 👍

Reply:
@ton31337 I will upstream the patch in a couple of weeks